src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -66,16 +80,45 @@ private static void readPropertiesFromFile(String[] args) {
 		String propFileLocation;
 		if(args.length == 0) {
 			M_log.error("Command line arguments are not provided");
-			System.exit(0);
+			System.exit(1);
 		}
 		propFileLocation = args[0];
 		properties = Utils.getPropertiesFromURL(propFileLocation);
 		canvasToken = properties.getProperty(CANVAS_TOKEN);
 		canvasURL = properties.getProperty(CANVAS_URL);
 		esbURL=properties.getProperty(ESB_URL);
-		sisTermIdFromPropsFile = properties.getProperty(SIS_TERM_ID);
+		getTermsInfoFromProperties(properties);
 		checkForApiType(properties);
 	}
+	/*
+	 * The Terms that needs for course deletion are listed in the properties file. Each term has 2 piece of information 
+	 * term1=2030;04/06/15, the sisTermCode and the endDate of that term. We will check if the current system date is
+	 * past the endDate then that term becomes eligible for deleting the unused/unpublished courses that are created 
```

Comment:
Right.  To avoid confusion name and explain from the point of view of how
it will be used not about where the value may (or may not) come from.

On Thu, Oct 1, 2015 at 12:48 PM, Pushyami Gundala <notifications@github.com>
wrote:

> In src/main/java/edu/umich/tl/CourseDelete.java
> <https://github.com/tl-its-umich-edu/canvasCourseDeleteUtility/pull/4#discussion_r40936925>
> :
>
> >  		checkForApiType(properties);
> >  	}
> > +	/*
> > +	 * The Terms that needs for course deletion are listed in the properties file. Each term has 2 piece of information
> > +	 * term1=2030;04/06/15, the sisTermCode and the endDate of that term. We will check if the current system date is
> > +	 * past the endDate then that term becomes eligible for deleting the unused/unpublished courses that are created
>
> It's how ever BSA defines it what date he wants to put in. Their is lot of
> confusion about end date from different systems. Mpathways has their own
> term end date, canvas has some floating end dates to keep the courses alive
> in the canvas system. To me it is the date after which the courses will be
> consider for deletion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/canvasCourseDeleteUtility/pull/4/files#r40936925>
> .
>